canonical-websites/www.ubuntu.com
# Reduce number of redirect urls

## Done

Add 'catch all' redirects for insights, content, news and devices

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Make sure the old redirects work with the new versions

Fixes https://github.com/ubuntudesign/web-squad/issues/239
